docs/mongocli/command/mongocli-auth-login.txt

```diff
@@ -74,7 +74,7 @@ Examples
 
 .. code-block::
 
-   To start the interactive setup:
+   To start the interactive login with the MongoDB Ops Manager or Cloud Manager account:
```

Comment:
```suggestion
   To start the interactive login with Cloud Manager account:
```
I don't think this command is available for ops manager

Reply:
Updated, thank you!